app/src/main/java/a3/com/convo/adapters/CardAdapter.java

```diff
@@ -68,6 +72,24 @@ public void done(Page object, ParseException e) {
                 if (e == null) {
                     tvTopic.setText(object.getName());
 
+                    CountDownTimer timer = new CountDownTimer(Constants.CARD_TIME, Constants.TIMER_INTERVAL) {
+                        @Override
+                        public void onTick(long l) {
+                            tvTime.setText(
+                                    String.format(context.getResources().getString(R.string.timer_format),
+                                            TimeUnit.MILLISECONDS.toMinutes(l),
+                                            TimeUnit.MILLISECONDS.toSeconds(l)
+                                                    - TimeUnit.MINUTES.toSeconds(TimeUnit.MILLISECONDS.toMinutes(l)))
+                            );
+                        }
+
+                        @Override
+                        public void onFinish() {
+                            tvTime.setText(R.string.next_card);
```

Comment:
Did you test this line? Shouldn't you be using something like this:
`String.format(context.getResources().getString(R.string.timer_format)`

Reply:
right, thank you!